File: php/src/TurnTicketDispenser/TurnTicket.php
<?php

declare(strict_types=1);

namespace RacingCar\TurnTicketDispenser;

class TurnTicket
{
    public function __construct(
        private int $turnNumber
    ) {
    }

    public function getTurnNumber(): int
    {
        return $this->turnNumber;
    }
}

File: CSharp/TurnTicketDispenser/TurnTicket.cs
namespace TDDMicroExercises.TurnTicketDispenser
{
    public class TurnTicket
    {
        private readonly int _turnNumber;

        public TurnTicket(int turnNumber)
        {
            _turnNumber = turnNumber;
        }

        public int TurnNumber
        {
            get { return _turnNumber; }
        }
    }
}